Core/Steps/SubSteps/ReleaseVersionAndMoveIssuesSubStep.cs

```diff
@@ -66,99 +75,67 @@ public void Execute (SemanticVersion currentVersion, SemanticVersion nextVersion
       AddNewlyReleasedVersionToClosedIssuesOnlyAssociatedWithFullVersion(currentVersion);
   }
 
+  private bool ShouldMoveIssuesToNextVersion (string versionID, string nextVersionID, out IReadOnlyList<JiraToBeMovedIssue> issuesToMove)
+  {
+    if (versionID == nextVersionID)
+    {
+      issuesToMove = Array.Empty<JiraToBeMovedIssue>();
+      return false;
+    }
+
+    issuesToMove = JiraIssueService.FindAllNonClosedIssues(versionID);
+    if (issuesToMove.Count == 0)
+      return false;
+
+    Console.WriteLine("These are some of the issues that will be moved by releasing the version on jira:");
+    PrintIssueListing(issuesToMove);
+    Console.WriteLine("Do you want to move these issues to the new version and release the old one or just release the old version?");
+
+    return InputReader.ReadConfirmation();
+  }
+
   private void AddNewlyReleasedVersionToClosedIssuesOnlyAssociatedWithFullVersion (SemanticVersion currentVersion)
   {
     var currentFullVersion = currentVersion.GetCurrentFullVersion().ToString();
     try
     {
-      var allJiraVersionsStartingWithFullVersion = _jiraVersionCreator.FindAllVersionsStartingWithVersionNumber(currentFullVersion);
+      var allJiraVersionsStartingWithFullVersion = JiraVersionCreator.FindAllVersionsStartingWithVersionNumber(currentFullVersion);
 
       if (allJiraVersionsStartingWithFullVersion.Count == 0)
         throw new UserInteractionException($"Could not find versions starting with '{currentFullVersion}' in JIRA.");
 
       var currentFullJiraVersion =
-          _jiraVersionCreator.FindVersionWithVersionNumber(currentFullVersion)
+          JiraVersionCreator.FindVersionWithVersionNumber(currentFullVersion)
           ?? throw new UserInteractionException($"Version '{currentFullVersion}' does not exist in JIRA.");
 
       var closedIssuesOnlyAssociatedWithFullVersion =
-          _jiraIssueService.FindIssuesWithOnlyExactFixVersion(allJiraVersionsStartingWithFullVersion, currentFullJiraVersion);
+          JiraIssueService.FindIssuesWithOnlyExactFixVersion(allJiraVersionsStartingWithFullVersion, currentFullJiraVersion);
 
       if (closedIssuesOnlyAssociatedWithFullVersion.Count == 0)
       {
         var message = $"Could not find any issues only associated with fixVersion '{currentFullVersion}', no moving of these issues is necessary.";
-        _console.WriteLine(message);
+        Console.WriteLine(message);
         _log.Debug(message);
         return;
       }
 
-      _console.WriteLine($"The following Jira tickets are closed and only associated with version '{currentFullVersion}':");
+      Console.WriteLine($"The following Jira tickets are closed and only associated with version '{currentFullVersion}':");
```

Comment:
We got some more jira/Jira casings. do you want to fix them?

Reply:
Fixed the ones I found that are close to the already done changes, might need a cleanup task